TSB Rev1 New Parts

ckaminsky7 opened the pull request:
# Pull Request (PR) into Circuits-Support-2022

## Part Description
Main Library Changes:
- 4 Pin Connector
- 3 Pin Connector (5 pin with two pins taken out)
- 5 Pin Connector (6 pin with one pin taken out)
- 5 Pin Connector (7 pin with two taken out)
- 8 Pin Connector
- Resistor(0603 -> 1.62, 13k, 33.2k)
- Polar Cap(3600uF)
- Inductor(PA4334 -> 3.6uH; WE-LMHI -> 56 uH)
- Shottky Diode(fixed anode and cathode orientation)

Temporary Library Changes:
- ADM825
- DCM4623
- NC Solid State Relay
- NO Solid State Relay
- Grounded Resistor(B20 -> 1.25k, 10k)

## Additional Information
All datasheets are linked in the descriptions in EAGLE.

## Checklist
- [ ] Did you create any new schematics or boards?
- - [ ] Did you *make a PR* for them in `circuits-2022`? If so, please pause until you get this PR merged.
- [x] Did you pull `main` into your branch?
- - [x] Did you *check for merge conflicts*?
- - [ ] Did you *resolve* any that occurred? ***If you are having trouble or are confused, contact a lead!***
- [x] Did you fill out the above template?
- [x] Did you assign the right people for review (on the right)?
- [x] Did you comply with the library style guidelines?

zli909 committed:
Added 3.32k and 8.66k 0603 res

zli909 committed:
Added addition .22uF 10V cap and 75k res

zli909 committed:
added 250V 1uF cap

zli909 committed:
added 8.35k resistor

ckaminsky7 committed:
Added resistors, inductors, capacitors, mic806, dcm4768, and nc ss optoisolator

ckaminsky7 committed:
Merge branch 'New-resistors' into DCDC_Converter_Rev_2

ckaminsky7 committed:
Added footprints to past added parts

ckaminsky7 committed:
Modified devices for the TSB

ckaminsky7 committed:
Added 0.1 uF, added connectors with distance

joshkirsh720 committed:
add normally open solid state relay

joshkirsh720 committed:
change to aqy214eha no solid state relay

ckaminsky7 committed:
Added adm825, fixed connectors

ckaminsky7 committed:
Merge branch 'DCDC_Converter_Rev_2' of https://github.com/hytech-racing/circuits-support-2022 into DCDC_Converter_Rev_2

ckaminsky7 committed:
Added shunt resistor

ckaminsky7 committed:
Added resistors and timer part necessary for the tsb

ckaminsky7 committed:
Added dip-4 footprint, more resistors, and fixed a 3 pin connector

joshkirsh720 committed:
change optocopuler part number

ckaminsky7 committed:
fixed ts-fused supply symbol

ckaminsky7 committed:
fixed b20 resistor footprint

ckaminsky7 committed:
Started to move parts to be grouped together

ckaminsky7 committed:
fixed diode footprint

zli909 committed:
added 0805 12.4 ohm resistor

ckaminsky7 committed:
Made temp library and transferred parts over

ckaminsky7 committed:
Merge branch 'DCDC_Converter_Rev_2' of https://github.com/hytech-racing/circuits-support-2022 into DCDC_Converter_Rev_2

ckaminsky7 committed:
Fixed parts in the main demo library

ckaminsky7 committed:
Added descriptions for all new parts and created devices of new parts

ckaminsky7 committed:
Fixed library issues

ckaminsky7 committed:
More final changes to library

19etweinstock reviewed (changes requested):
Missing documentation for B20 footprint (need resistor size and mount)
Missing 750 ohm resistor variant
Manufacturer on ltc6994-1 is wrong (Analog Electronics is not a company)
ltc6994 device and symbol should be in temp library
ltc6994 symbol needs description and datasheet link
ltc6994 device needs better description
check current ratings for pa4334 inductor
new inductor footprints need datasheet
molex footprint and plug size link to dual row variant
parts duplicated in temp library and regular library
dip4 and dip4 air-gap packages need footprint datasheets
adm823 should not be removed from the library as it is used on the acu
dcm4623 should not be in main library
mic806 symbol should have been deleted (i did this)
ltc6994 should have been in temp library (name should have been LTC6994-* rather than LTC6994_* to better reflect actual name)
resistor ground should not be in main library
630V 0.1uF, 250V 0,1uF capacitor does not use greek mu symbol
1A fuse has wrong MPN and no constant (i added this for AMS so please remove)
MB fuse holder shoulder reference physical fuse part. please add the 1A fuse part to hytech externals
voltage continuous on resistors should always have two decimal points unless it hits max voltage
tvs diodes need constant attributes
connectors need full 4 attributes
inductor attributes constant
greek letters on inductor values
capacitor polar should be capacitor polarized. greek letters.
new stuff should not be in main library

19etweinstock committed:
Started fixing some things

ckaminsky7 committed:
Added 750 ohm B20 resistor

ckaminsky7 committed:
Fixed device name of LTC6994

ckaminsky7 committed:
14 pin connectors (#83)

* Fixed Yageo links (#42)

* Fixed Yageo links

* Added TS+5V

* Dru fix (#43)

* Removed old fabs

* Adjusted DRC names

* Fixed DRC values

* Updated 1/2 layer defaults

* Fixed 1/2 defaults

* Fixed 4/6 defaults

* Danger high voltage (#44)

* Added "DANGER HIGH VOLTAGE" text

* Fixed wrong layers

* Fixed all text to place layers

Co-authored-by: noahsb1 <71232430+noahsb1@users.noreply.github.com>
Co-authored-by: 19etweinstock <ethangweinstock@gmail.com>

* Current sensor board (#45)

* Created symbol and device

* Edited Current Sensor Pins

* Made requested changes to Current Sensor per Pull Request

Narrowed symbol width, changed IN+ and IN- pins to IO, made them evenly spaced, renamed device and added attribute for a specific technology variation, capitalized "D" in datasheet in description, added prefix.

* Edited attributes

* Merged Temp library with main library

* Added nanofit and microfit connectos 14 pin

* Added new parts

* Added 14 pin connectors

Co-authored-by: 19etweinstock <30483731+19etweinstock@users.noreply.github.com>
Co-authored-by: Arvind Srinivasan <arvindsrinivasan@gatech.edu>
Co-authored-by: noahsb1 <71232430+noahsb1@users.noreply.github.com>
Co-authored-by: 19etweinstock <ethangweinstock@gmail.com>
Co-authored-by: Steven Elias <91804237+selias8@users.noreply.github.com>

ckaminsky7 committed:
Merge branch 'main' into DCDC_Converter_Rev_2

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+</connects>
+<technologies>
+<technology name="">
+<attribute name="DKPN" value="255-2897-ND" constant="no"/>
+<attribute name="MANUFACTURER" value="Panasonic Industrial Devices" constant="no"/>
+<attribute name="MOPN" value="769-AQY414EHA" constant="no"/>
+<attribute name="MPN" value="AQY414EHA" constant="no"/>
```
constant

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+</connects>
+<technologies>
+<technology name="">
+<attribute name="DKPN" value="255-2897-ND" constant="no"/>
+<attribute name="MANUFACTURER" value="Panasonic Industrial Devices" constant="no"/>
+<attribute name="MOPN" value="769-AQY414EHA" constant="no"/>
+<attribute name="MPN" value="AQY414EHA" constant="no"/>
+</technology>
```
constant

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<deviceset name="RELAY_SOLIDSTATE_NO_*" prefix="K">
+<description>Normally Open Solid-State Relay
+&lt;br&gt;
+&lt;a href="file:///C:/Users/kamin/Downloads/semi_eng_ge1a_aqy21_e.pdf"&gt;Datasheet&lt;/a&gt;</description>
```
links to a local file i cannot access

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+</connects>
+<technologies>
+<technology name="">
+<attribute name="DKPN" value="" constant="no"/>
+<attribute name="MANUFACTURER" value="" constant="no"/>
+<attribute name="MOPN" value="" constant="no"/>
+<attribute name="MPN" value="" constant="no"/>
+</technology>
```
constant

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+</connects>
+<technologies>
+<technology name="">
+<attribute name="DKPN" value="255-1436-5-ND" constant="no"/>
+<attribute name="MANUFACTURER" value="Panasonic Inustrial Devices" constant="no"/>
+<attribute name="MOPN" value="769-AQY214EHA" constant="no"/>
+<attribute name="MPN" value="AQY214EHA" constant="no"/>
```
constant

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+</device>
+</devices>
+</deviceset>
+<deviceset name="LTC6994_*" prefix="U">
```
- not _

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<attribute name="DKPN" value="505-LTC6994CS6-1#TRMPBFTR-ND"/>
+<attribute name="MANUFACTURER" value="Analog Electronics"/>
+<attribute name="MOPN" value="584-C6994CS6-1TRMPBF"/>
+<attribute name="MPN" value="LTC6994CS6-1#TRMPBF" constant="no"/>
```
constant

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<pin name="SET" x="-2.54" y="-5.08" length="short"/>
+<pin name="GND" x="-2.54" y="-12.7" length="short" direction="pwr"/>
+<pin name="OUT" x="17.78" y="-2.54" length="short" rot="R180"/>
+<pin name="V+" x="17.78" y="-5.08" length="short" direction="pwr" rot="R180"/>
```
VDD

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<wire x1="10.16" y1="-2.54" x2="17.78" y2="-2.54" width="0.1524" layer="94"/>
+<wire x1="12.7" y1="-10.16" x2="17.78" y2="-10.16" width="0.1524" layer="94"/>
+</symbol>
+<symbol name="LTC6994">
```
missing datasheet in description

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+</layers>
+<library>
+<packages>
+<package name="B20">
```
missing datasheet in description

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<rectangle x1="21.19" y1="-10.25" x2="25.19" y2="10.25" layer="40"/>
+<rectangle x1="-25.19" y1="-10.25" x2="-21.19" y2="10.25" layer="40"/>
+</package>
+<package name="DIP-4">
```
missing datasheet in description

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<circle x="-3.175" y="-3.81" radius="0.381" width="0" layer="21"/>
+<rectangle x1="-3.556" y1="-6.096" x2="3.556" y2="6.096" layer="39"/>
+</package>
+<package name="DIP-4_AIRGAP">
```
missing datasheet in description
you don't have all the right layers for airgap

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<package name="SOT-23-6">
+<description>6-Lead Plastic ThinSOT-23
+&lt;br&gt;
+&lt;a href="https://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/ltc-legacy-sot-23/SOT_6_05-08-1636.pdf"&gt;Footprint Detailing&lt;/a&gt;</description>
```
should just say Footprint

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<gate name="G$1" symbol="RELAY_SOLIDSTATE_NO" x="0" y="0"/>
+</gates>
+<devices>
+<device name="" package="DIP-4">
+<connects>
+<connect gate="G$1" pin="CONTACT1" pad="3"/>
+<connect gate="G$1" pin="CONTACT2" pad="4"/>
+<connect gate="G$1" pin="CONTROL+" pad="1"/>
+<connect gate="G$1" pin="CONTROL-" pad="2"/>
+</connects>
+<technologies>
+<technology name="">
+<attribute name="DKPN" value="" constant="no"/>
+<attribute name="MANUFACTURER" value="" constant="no"/>
+<attribute name="MOPN" value="" constant="no"/>
+<attribute name="MPN" value="" constant="no"/>
```
missing attributes for DIP-4 footprint

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<deviceset name="RESISTOR_GND_*" prefix="R">
+<description>20W Axial Through Hole Resistor
+&lt;br&gt;
+&lt;a href="https://www.ohmite.com/assets/docs/res_200.pdf?r=false"&gt;Datasheet&lt;/a&gt;</description>
```
not mouser link

19etweinstock commented on EAGLE Libraries/HyTechDevicesTempTSB.lbr:
```diff
@@ -0,0 +1,626 @@
+<deviceset name="RELAY_SOLIDSTATE_NC_*" prefix="K">
+<description>Normally Closed Solid-State Relay
+&lt;br&gt;
+&lt;a href="file:///C:/Users/kamin/Downloads/semi_eng_ge1b_aqy41_e%20(1).pdf"&gt;Datasheet&lt;/a&gt;</description>
```
link to local file

ckaminsky7 committed:
Update HyTechDevices.lbr

ckaminsky7 committed:
Merge branch 'main' into DCDC_Converter_Rev_2

ckaminsky7 committed:
Update HyTechDevices.lbr

ckaminsky7 committed:
Update HyTechDevices.lbr

ckaminsky7 committed:
Update HyTechDevices.lbr

ckaminsky7 committed:
Fixed xml issues

ckaminsky7 committed:
Added Rohm resistors and no relay

ckaminsky7 committed:
Fixed wrong parts, added hv resistors, added 8 pin header

ckaminsky7 committed:
Added ordering components, shunt ressitors

ckaminsky7 committed:
Added female header external and flex stack external

ckaminsky7 committed:
Fixed externals

ckaminsky7 committed:
Fixed molex connectors

ckaminsky7 committed:
Made changes Ethan suggested

ckaminsky7 committed:
Made final changes to libraries

ckaminsky7 committed:
Merge branch 'main' into DCDC_Converter_Rev_2

ckaminsky7 committed:
fixed parts in the libraries

ckaminsky7 committed:
Added hv resistors

ckaminsky7 committed:
Added resistors

ckaminsky7 committed:
added 4 pin molex part back

ckaminsky7 committed:
Merge branch 'main' into DCDC_Converter_Rev_2
